src/OrbitGl/App.h

```diff
@@ -645,6 +645,11 @@ class OrbitApp final : public DataViewFactory,
   absl::flat_hash_map<std::pair<std::string, std::string>, orbit_base::Future<ErrorMessageOr<void>>>
       symbols_currently_loading_;
 
+  // Set of modules where an symbol loading error has occurred. The module identifier is file path
+  // and build ID.
+  // ONLY access this from the main thread.
+  absl::flat_hash_set<std::pair<std::string, std::string>> modules_with_symbol_loading_error_;
```

Comment:
What about defining a struct for this? Just from looking at this I have to guess what those two strings are.

Reply:
added the bug TODO

Reply:
For the sake of completeness:

```cpp
struct ModuleIdentifier {
  std::string file_path;
  std::string build_id;
  
  friend bool operator==(const ModuleIdentifier& lhs, const ModuleIdentifier& rhs) { return
  std::tie(lhs.file_path, lhs.build_id) == std::tie(rhs.file_path, rhs.build_id); }
  friend bool operator!=(const ModuleIdentifier& lhs, const ModuleIdentifier& rhs) { return !(lhs == rhs); }
  template <typename H>
  friend H AbslHashValue(H h, const ModuleIdentifer& id) {
    return H::combine(std::move(h), id.file_path, id.build_id);
  }
```